package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
package.json is metadata for the project, not just the gulp file. Should reflect the main repo.